client/components/App.jsx

```diff
@@ -1,13 +1,50 @@
 import React from 'react';
+import MovieInfo from './MovieInfo.jsx';
+import MoviePoster from './MoviePoster.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      movieId: 1,
+      poster: ''
+    };
   }
+
+  // fetch movie poster on mount
+  componentDidMount() {
+    this.getMoviePoster(this.state.movieId);
+  }
+  getMoviePoster(id) {
```

Comment:
Does your GET request return a single movie poster from the DB based on id, or does it return all the movie posters and then filter through them on the frontend?

Reply:
The GET request fetches all movie posters and filtering is done on the front-end. I tried to implement a GET request with some data passed in to target only one movie poster but it would not allow me. If you know of a better method to solve this please let me know.